src/clj/puppetlabs/services/file_serving/config/puppet_fileserver_config_core.clj

```diff
@@ -0,0 +1,93 @@
+(ns ^{:doc
+      "Tools to parse Puppet fileserver.conf."}
+    puppetlabs.services.file-serving.config.puppet-fileserver-config-core
+  (:require
+    [puppetlabs.kitchensink.core :as ks]
+    [schema.core :as schema]
+    [puppetlabs.puppetserver.ringutils :as ring])
+  (:use [clojure.java.io :only (reader)]
```

Comment:
Since Clojure 1.4.0, you've been able to pass a `:refer` option to `:require`. I believe it's now considered best practice to do this instead of `:use`.